utils/html_functions.py

```diff
@@ -51,7 +54,10 @@ def body_to_html(self, body, savings):
         for result in body:
             html = html + '<tr>'
             for i in result:
-                html = html + "<td>%s</td>" % (i)
+                if result.index(i) == 9:
```

Comment:
Add some comments here - What is at index 9? and why are we skipping it?
Do we have a better way of checking this condition? If I remove the item at index 3 in the future, this will break, right? Because now the item at index 9 is at index 8.

Reply:
Now I have added the comment there I'll push it in next PR.
And as you asked to check if we can add another condition then my answer for this question is - we don't have same criteria for all the AWS resources or services so I think it's might be difficult to add condition in different way.